source/_integrations/light.mqtt.markdown

```diff
@@ -775,7 +775,12 @@ light:
 The `mqtt` light platform with template schema lets you control a MQTT-enabled light that receive commands on a command topic and optionally sends status update on a state topic.
 It is format-agnostic so you can use any data format you want (i.e., string, JSON), just configure it with templating.
 
-This schema supports on/off, brightness, RGB colors, XY colors, color temperature, transitions, short/long flashing and effects.
+This schema supports on/off, brightness, RGB colors, XY colors, white color temperature, transitions, short/long flashing and effects. Following units and value ranges are passed to/obtained from the system.
+* RGB colors, values from a range: 0 - 255
+* Brightness, value from a range: 0 - 255
+* White color temperature: Mireds.
```

Comment:
These are better fitting in the configuration section with their respective configuration options.

Reply:
I was considering that, but it would require to repeat the same information several times.

IMO the best would be reformat whole document, to begin with system expectations. But I would't touch it for obvious reasons. 

May it stay or you insist to move it to config section, mentioning it everytime it's needed?

Reply:
> or you insist to move it to config section

Yes.